dev/io.openliberty.security.mp.jwt.2.1.config/src/io/openliberty/security/mp/jwt/v21/config/impl/MpConfigProxyServiceImpl.java

```diff
@@ -70,9 +58,32 @@ public String getVersion() {
         return MP_VERSION;
     }
 
+    /**
+     * @return
+     */
+    @Sensitive
+    @Override
+    public <T> T getConfigValue(ClassLoader cl, String propertyName, Class<T> propertyType) throws IllegalArgumentException, NoSuchElementException {
```

Comment:
With change we now have two identical `getConfigValue()` methods. It'd be nice to come back and rework this a bit so we only need one.

Reply:
I agree with this.  I think this can be reworked.  Today com.ibm.ws.security.mp.jwt bundles doesn't import MP Config features so that is why I didn't do an AbstractMpConfigProxyService class or do a default impl in the interface in there.  If that was changed to allow that bundle to import the mp config packages, then that would be an easy thing to do.  I will probably work with Joe some to rework this anyway since it performs poorly the way it is currently coded even before my changes.